pkg/registry/debian/debian.go

```diff
@@ -292,25 +293,37 @@ func (r HTTPRegistry) ArtifactURL(ctx context.Context, name, artifact string) (s
 		return "", err
 	}
 	var response fileInfo
-	fileinfoURL := urlx.Copy(snapshotURL)
-	{
-		fileinfoURL.Path += path.Join("mr/package", name, a.Version.BinaryIndependentString(), "binfiles", a.Name, a.Version.String())
-		query := fileinfoURL.Query()
-		query.Add(
-			"fileinfo",
-			"1",
-		)
-		fileinfoURL.RawQuery = query.Encode()
-	}
-	{
-		r, err := r.get(ctx, fileinfoURL.String())
-		if err != nil {
+	var fileinfoURL *url.URL
+	for _, epoch := range []string{"", "1"} {
```

Comment:
would you be able to add a comment on what an epoch is and why we try two of them?